cbor2/types.py

```diff
@@ -62,36 +64,30 @@ def __repr__(self):
         return 'CBORTag({self.tag}, {self.value!r})'.format(self=self)
 
 
-class CBORSimpleValue(object):
+class CBORSimpleValue(namedtuple('CBORSimpleValue', ['value'])):
     """
     Represents a CBOR "simple value".
 
     :param int value: the value (0-255)
     """
 
-    __slots__ = 'value'
+    __slots__ = ()
+    __hash__ = namedtuple.__hash__
```

Comment:
Is this really necessary? The class already inherits from namedtuple so wouldn't that implementation of `__hash__()` be used anyway?

Reply:
Surprisingly, yes if you redefine `__eq__` on a namedtuple subclass you have to.